test/test.js

```diff
@@ -68,30 +80,30 @@ describe('get-window', function () {
   }
 
   it('should work with the child node of an IFRAME element', function () {
-    var iframe = document.createElement('iframe');
-    document.body.appendChild(iframe);
+    var iframe = doc.createElement('iframe');
+    doc.body.appendChild(iframe);
 
     // `contentWindow` should be used for best browser compatibility
-    var doc = getDocument(iframe.contentWindow);
-    assert.equal(9, doc.nodeType);
+    var iframeDoc = getDocument(iframe.contentWindow);
```

Comment:
Here, `doc` was already taken, so I took the liberty to change it to `iframeDoc`.